torch/_C/_profiler.pyi

```diff
@@ -58,7 +58,6 @@ class _ExperimentalConfig:
         profiler_measure_per_kernel: bool = ...,
         verbose: bool = ...,
         performance_events: List[str] = ...,
-        privateuse1_config: Dict = ...,
```

Comment:
I don't think this is a valid param to the constructor:

https://github.com/pytorch/pytorch/blob/0d6d97d956690b16ba5224f8accf748b90d16108/torch/csrc/profiler/python/init.cpp#L265-L273